requirements.txt

```diff
@@ -6,4 +6,6 @@ numba
 sciris
 scirisweb
 gunicorn
-plotly_express
\ No newline at end of file
+plotly_express
+urllib3
+requests
```

Comment:
If it's a script we don't need to add these to the requirements

Reply:
You keep saying script, do you mean a bash script? If we are using python for the script, these requirements make http request pretty simple and add in a bunch of helpers to simplify things like making requests for json.

Reply:
i mean python script, i.e. run `python pull_demographic_data.py` as opposed to
```
import covasim as cv
cv.datasets.pull_demographic_data()
```

Reply:
That exists python covasim/datasets/update_sources.py

Reply:
It is now part of the Docker build and the setup.py call.